### ISSUE TITLE
some error caused by the last release 

### ISSUE THREAD

**Issue by freestyle21** [faisalman/ua-parser-js#269]
some warning show in my app caused by the last release `0.7.16 ` in npm 

```
WARNING  Compiled with 1 warnings                                      

 warning  in ./node_modules/ua-parser-js/src/ua-parser.js

Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

```

I found some package rely on `ua-parser-js` from the package-lock.json, and I install `ua-parse-js@0.7.15`, everything goes well.

**Comment by Ttdnts** [comment]
It seems fixed in 0.7.17, I updated to the latest version and it works well

**Comment by faisalman** [comment]
Duplicate #268
